### PR TITLE
Added Logic for retrieving points

### PR DESCRIPTION
**Pull request by MishkaSorochinskiy**

### CODE REVIEW

File: DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            this._unitofwork = unitofwork;
+        }
+
+       public Task<PointsViewModelResult> Handle(CommandExecutionContext executionContext, GetPointsQuery request)
+        {
+            var perimeter= _unitofwork.PerimeterRepository.FindById(request.PerimeterId);
+
+            if (perimeter == null)
+                throw new NullReferenceException("Perimeter with specified id does not exist!");
```
Comment (VoitykRoman) [inline DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQueryHandler.cs:27]: Don`t throw NullReferenceExcpetion by yourself. i would like to recommend you create your custom exception class.

Comment (MishkaSorochinskiy) [reply DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQueryHandler.cs:27]: I left it for future

File: DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQuery.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DeratControl.Application.Interfaces;
+
+namespace DeratControl.Application.Points.Queries.GetPointsByPerimeter
+{
+    public class GetPointsQuery:IRequest
```
Comment (SviatoslavMykhailiv) [inline DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQuery.cs:8]: spaces before and after :

File: DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQueryHandler.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        private IUnitOfWork _unitofwork;
+
+        public GetPointsQueryHandler(IUnitOfWork unitofwork)
+        {
+            this._unitofwork = unitofwork;
+        }
+
+       public Task<PointsViewModelResult> Handle(CommandExecutionContext executionContext, GetPointsQuery request)
+        {
+            var perimeter= _unitofwork.PerimeterRepository.FindById(request.PerimeterId);
```
Comment (SviatoslavMykhailiv) [inline DeratControl.Application/Points/Queries/GetPointsByPerimeter/GetPointsQueryHandler.cs:24]: FindById should be awaitable

**Review by SviatoslavMykhailiv — CHANGES_REQUESTED** [review]

**Pull request merged**